### DISCUSSION TITLE
How to run saved model with new data?

### DISCUSSION THREAD

**Discussion by MinSukMoon** [RUCAIBox/RecBole#1312] [Q&A]
Hello, 

I am just wondering how to run saved model with new data. 
If it is possible to run saved model with new data please explain how to do it.

Thank you in advance :)

**Comment by Ethan-TZ** [comment]
@MinSukMoon Hello, thanks for your attention to RecBole!
Do you want to train saved model with a new dataset? It is not supported since we can not ensure the user or item set to be the same with original data and the same external ID is remap to same internal ID.
If you want to just predict the new data where each user or item has appeared on the original dataset, the **case_study** module is recommended. Specially, **Recbole/run_example/case_study_example.py** provides a complete usage of the interface. Meanwhile, you can refer to [Case Study](https://recbole.io/docs/user_guide/usage/case_study.html) for more help.

**Reply by adhiiisetiawan** [reply]
Hi @Ethan-TZ i have similar question, how to predict the trained model in RecBole? i already train using my data but still can't find how to predict/inference for my model. Is there any resource or documentation to do predict/inference?
